com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Changes in Alembic for Unity
 ## [2.0.0-preview.1] - 2019-08-07
 ### Changes
-- Introduced public API for Alembic playback and recording. Bumped minimum Unity
-  version to 2019.2
+- Minimum unity version is  2019.3.
+- Introduced public API for Alembic playback and recording. 
+- Multithread the loading of alembic files.
+- Removed the LitAlembic HDRP shader. The same features are supported by HDRP native shaders enabling 'Additional Velocity Change'.
```

Comment:
Removed the LitAlembic HDRP shader that has been obsoleted by HDRP native shaders that now offers the once missing velocity option.